spec/safe_statements_spec.rb

```diff
@@ -1256,6 +1256,59 @@ def up
           end
         end
 
+        describe "safe_add_index_on_empty_table" do
+          it "creates index when table is empty" do
+            setup_migration = Class.new(migration_klass) do
+              def up
+                unsafe_create_table :foos
+                unsafe_add_column :foos, :bar, :text
+              end
+            end
+
+            setup_migration.suppress_messages { setup_migration.migrate(:up) }
+
+            test_migration = Class.new(migration_klass) do
+              def up
+                safe_add_index_on_empty_table :foos, [:bar]
+              end
+            end
+
+            expect do
+              test_migration.suppress_messages { test_migration.migrate(:up) }
+            end.to make_database_queries(matching: /CREATE INDEX "index_foos_on_bar"/, count: 1)
+
+            indexes = ActiveRecord::Base.connection.indexes("foos")
+            expect(indexes.size).to eq(1)
+            expect(indexes.first).to have_attributes(:table => "foos", :name => "index_foos_on_bar", :columns => ["bar"])
```

Comment:
Nit:
```suggestion
            expect(indexes.first).to have_attributes(table: "foos", name: "index_foos_on_bar", columns: ["bar"])
```

Reply:
Addressed in https://github.com/braintree/pg_ha_migrations/pull/93/commits/e668a5c0311bc26dbfcdfca881ed401d469b8f86